docs/contributing/running-tests.md

```diff
@@ -76,3 +76,20 @@ To execute all the instrumentation tests runnable as GraalVM native executables:
 Some of the instrumentation tests (and all of the smoke tests) spin up docker containers via
 [testcontainers](https://www.testcontainers.org/). If you run out of space, you may wish to prune
 old containers, images and volumes using `docker system prune --volumes`.
+
+# Troubleshooting CI Test Failures
+
+CI test logs are pretty big around 75MB. To make it easier to troubleshoot test failures, you can download the raw logs from the CI job and then look for
+`Publishing build scan...` in the logs. Copy the URL from the logs and open it in a browser. This will give you a nice view of the test execution breakdown.
+
+## How to download the raw logs
+
+1. Click on the `Details` link in the CI job under `Some check were not sussessful` section of your PR.
+2. Click on one of the failed tests in the left panel.
+3. Click on the `Setting` gear in the top right corner of the logs panel and select 'View raw logs'.
+4. While the page is loading, you can right click to save the page as a text file locally.
```

Comment:
easier to right click on 'View raw logs' and download directly

Reply:
are you referring to 
![image](https://github.com/open-telemetry/opentelemetry-java-instrumentation/assets/56097766/6c194065-2b55-4178-b18b-43b5ce1d421f)

i think we want to save file as, not the link. there is no option on right click to save file as?